openmdao/visualization/n2_viewer/index.html

```diff
@@ -1,83 +1,206 @@
+<head>
+    <script src="https://kit.fontawesome.com/1b89a70380.js" crossorigin="anonymous"></script>
```

Comment:
I think previously we have avoided requiring internet access to use the viewer (by having local copies of dependencies like d3, etc.).